code/game/machinery/computer/robot_control.dm

```diff
@@ -142,6 +143,7 @@
 	data["can_hack"] = can_hack_any(user)
 	data["cyborgs"] = list()
 	data["safety"] = safety
+	data["detonate_cooldown"] = (round((detonate_cooldown - world.time) / 10))
```

Comment:
This one should work how you want it too, but you should check. Also this is just my opinion, but the second set of parenthesis made this harder to read.
```suggestion
	data["detonate_cooldown"] = round(COOLDOWN_TIMELEFT(src, detonate_cooldown) / 10)
```